src/components/panel/panel.js

```diff
@@ -1188,6 +1211,24 @@ MdPanelRef.prototype._addStyles = function() {
 
 
 /**
+ * Update the position configuration of a panel
+ * @param {MdPanelPosition} position
+ */
+MdPanelRef.prototype.updatePosition = function(position) {
```

Comment:
You need to add this documentation also to the public documentation above, probably with a description of why you someone would use this. (~line 330)